Cortes-Zuleta/template/simple_functions.py

```diff
@@ -10,3 +10,10 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
+    if value % 2.0 == 0 or value % 3.0 == 0 or value % 5.0 == 0:
```

Comment:
this condition does not seem to be enough to define whether a number is prime or not, it would work only for the first few prime numbers. For instance 121 = 11 x 11 would be marked as 'prime' while is actually not.
You should revise it to include all possibile prime numbers, e.g.
      for i in range(2,value) : 
               if not (value % i) : return False
               else : return True